ur_robot_driver/doc/resources/README.md

```diff
@@ -0,0 +1,30 @@
+# Resources about ROS 2 Universal Robot driver
+
+The files in this folder are available for use under [![License](https://i.creativecommons.org/l/by/4.0/88x31.png)](http://creativecommons.org/licenses/by/4.0/) license.
```

Comment:
This image looks a little awkward when inline with text. Can we just saw Creative Common license (and maybe move the image to below the text?)

Reply:
Why Creative Common license?

Reply:
The idea is to encourage people to use the information from the presentation, but they have to name us on each use.